examples/non-fungible-token/tests/workspaces/test_approval.rs

```diff
@@ -122,15 +117,12 @@ async fn simulate_approval_with_call(
 
     let res = nft_contract
         .call("nft_approve")
-        .args_json((
-            TOKEN_ID,
-            approval_receiver_contract.id(),
-            Some("return-now".to_string()),
-        ))
+        .args_json((TOKEN_ID, approval_receiver_contract.id(), Some("return-now".to_string())))
         .max_gas()
-        .deposit(NearToken::from_yoctonear(450000000000000000000))
+        .deposit(NearToken::from_yoctonear(550000000000000000000))
         .transact()
         .await?;
+    assert!(res.is_success(), "{}", res.into_result().unwrap_err().to_string());
```

Comment:
Nit: this looks like the best way to assert in all other spots of unwrap-s, printing error details in case of the assertion fails